### ISSUE TITLE
Set up on-platform production service for Bigpicture landing pages 

### ISSUE THREAD

**Issue by yohell** [NBISweden/aida-data-hub#304]
We currently use GitHub Pages. That might not be the best system for this, for example because it is in Microsoft Azure, meaning that if a contributor makes a mistake and contributes information they did not intend to contribute, this information will end up on Microsoft Azure. 

We should probably run our own service on-platform instead. Maybe just a static site generator (such as Hugo?) could be good for this? (At least at this stage? Or are there good reason to choose another technology?) 

The service should have a good domain name (such as datasets.bp.nbis.se?) and certificate handling set up.

This service should ideally accept a file from the dataset, not change anything in the file, and put up a corresponding pretty page on the website.

**Comment by pontus** [comment]
I'm not sure I'm totally convinced by the benefits of this (given that it's intended for public consumption and that should be very strongly communicated), but if we do it for whatever reason it probably makes the most sense to have the generated landing pages (and possibly sources) in s3 bucket(s). That can then possibly be used to serve content directly or more likely served by a nginx or similar acting as proxy.

That still leaves the issue of getting the source page and generating content, but that seems reasonable to tie to the metadata submitter (and makes sense for a new card).

**Comment by yohell** [comment]
Yes, the exact factoring into however many services can of course be discussed, and this this card correspondingly broken down into smaller cards. And maybe some of the contextual text from this card (and from the #335 card that you created, and which I pulled into the same epic just now? :-) could be moved into the epic instead?
